### PR TITLE
Adding chart for dotted and solid lines in pipeline

### PR DESCRIPTION
**Pull request by Typositoire**
https://github.com/concourse/concourse/issues/731

Would that be correct as a start? People can make it prettier :p But at least there's a legend now.

image

The wording might not be the best one. Ideas are welcome!

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Typositoire Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/atc&pullRequestId=168)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/atc&pullRequestId=168) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Typositoire Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/atc&pullRequestId=168)!

File: web/elm/src/Pipeline.elm [hunk 1]
```diff
@@ -234,6 +234,10 @@ view model =
             , Html.dd [] [ Html.text "aborted" ]
             , Html.dt [ class "paused" ] []
             , Html.dd [] [ Html.text "paused" ]
+            , Html.dt [ class "dotted" ] [ Html.text "." ]
+            , Html.dd [] [ Html.text "Manually triggered job" ]
```
Comment (VanAxe) [inline web/elm/src/Pipeline.elm]: I suggest you respect the case of the elements above, this `, Html.dd [] [ Html.text "manually triggered job" ]`. Same goes for line `240`.

As for the language, I'm not sure `manually trigerred job` is appropriate for the following reasons :
- [One can disable manual triggers](https://concourse.ci/configuring-jobs.html#disable_manual_trigger), making this a misnomer.
- We're really talking about resources here.

I would suggest `dependency link` and `trigger dependency`... but I'm not even particularly excited about those suggestions.

Comment (chendrix) [inline web/elm/src/Pipeline.elm]: I tend to agree with @VanAxe here.

Comment (chendrix) [inline web/elm/src/Pipeline.elm]: How about

`dependency` and `dependency (trigger)`

Comment (VanAxe) [inline web/elm/src/Pipeline.elm]: Terser, I like it.

**Comment by chendrix** [comment]
Thanks again for the PR @Typositoire !

**Pull request closed without merging**